src/components/KTableView/KTableView.vue

```diff
@@ -547,6 +553,13 @@ const props = defineProps({
     type: Array as PropType<TableViewData>,
     default: () => [],
   },
+  /**
+   * A prop to pass row object key to be used as a unique identifier
```

Comment:
nit:
1. we don't need to explain it's a prop
2. reword
```suggestion
   * Provide the name of the data property keyto utilize as a unique identifier, or a function that receives the `row` object as a parameter that generates a unique identifier string for each row.
```